card.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from datetime import datetime
+from uuid import UUID
+
+from card_kind import Kind
+
+
+@dataclass
+class Card:
```

Comment:
Suggestion: Consider making it json serializable with default values (if it's not a big thing, if it is move it to another feature) to make the Card Generator simpler (or maybe even remove the generator)